inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -374,7 +374,7 @@ def execute_from_manifest(self,
 
     def _is_reboot_device(self) -> bool:
         return self._reboot_device not in ["No", "N", "n", "no", "NO"]
```

Comment:
**<h2>$\color{#FFA500}Improvements: CodeSimplification$</h2>**
**Suggestion:** Refactor the '_is_reboot_device' method to use a case-insensitive comparison for better reliability.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
    def _is_reboot_device(self) -> bool:
        return self._reboot_device.lower() not in ["no", "n"]
```